android/src/main/java/com/oney/WebRTCModule/DailyAudioManager.java

```diff
@@ -161,15 +158,13 @@ private void transitionOutOfCallMode() {
     // Does things in the reverse order of transitionOutOfCallMode()
     private void transitionToCurrentCallMode(Mode previousMode) {
         // Already in a call, so all we have to do is configure devices
-        if (previousMode == Mode.VIDEO_CALL || previousMode == Mode.VOICE_CALL) {
+        if (previousMode == Mode.VIDEO_CALL || previousMode == Mode.VOICE_CALL || previousMode == Mode.USER_DEFINED) {
```

Comment:
trying to remember why this check isn't simply `previousMode !== Mode.IDLE`...

Reply:
I believe It is to make sure we are receiving some valid `previousMode`, isn't It ?

Reply:
Like, some `previousMode` that our application will recognize.